kubernetes/charts/weblogic-operator/values.yaml

```diff
@@ -117,10 +117,19 @@ elasticSearchHost: "elasticsearch.default.svc.cluster.local"
 # This parameter is ignored if 'elkIntegrationEnabled' is false.
 elasticSearchPort: 9200
 
-# javaLoggingLevel specifies the Java logging level for the operator.
+# javaLoggingLevel specifies the Java logging level for the operator. This affects the operator pod's
+# log output and the contents of log files in the container's /logs/ directory.
 # Valid values are: "SEVERE", "WARNING", "INFO", "CONFIG", "FINE", "FINER", and "FINEST".
 javaLoggingLevel: "INFO"
 
+# javaLoggingFileSizeLimit specifies the maximum size in bytes of the Java logging files in the operator container's
+# /logs/ directory.
+javaLoggingFileSizeLimit: 2000000
```

Comment:
Current negotiations have this defaulting to 20MB.  That'd be 7 zeroes instead of 6..

Reply:
Yup... sigh. Thanks for checking.